src/main/resources/META-INF/plugin.xml

```diff
@@ -17,25 +17,28 @@
     <depends>com.intellij.modules.platform</depends>
     <depends>com.intellij.modules.java</depends>
     <extensions defaultExtensionNs="com.intellij">
+        <registryKey key="Developer Mode" description="Registry for showing id names suggesting plugin notifications"
```

Comment:
Usually registry key has `dot.separated.format`.

Look at the examples: 
<img width="807" alt="image" src="https://user-images.githubusercontent.com/7608535/100773179-92634600-3411-11eb-92ca-ed23d549f91b.png">. 
<img width="479" alt="image" src="https://user-images.githubusercontent.com/7608535/100773211-9c854480-3411-11eb-8ee3-ec2fac2772ce.png">